openreview/openreview.py

```diff
@@ -17,7 +18,7 @@ def __init__(self, baseurl=None, process_dir='../process/', webfield_dir='../web
             try:
                 self.baseurl = os.environ['OPENREVIEW_BASEURL']
             except KeyError:
-                self.baseurl = 'http://localhost:3000'
```

Comment:
I would keep the localhost by default, and force the user to specify the baseUrl.